fili-core/src/main/resources/moduleConfig.properties

```diff
@@ -54,10 +54,10 @@ bard__memcached_expiration_seconds = 3600
 # If using memcached, make sure this aligns with the setting memcached is using, overridden with it's -I parameter
 bard__druid_max_response_length_to_cache = 9223372036854775807
 
-# Data Cache strategy, which is one of the following
-# 1. ETagOnly
-# 2. LocalSignatureOnly (cache V2, needs the above flag set as well)
-# 3. TtlOnly (cache V1)
+# Data Cache strategy, whose value is one of the following
+# 1. ETag
+# 2. LocalSignature (cache V2)
+# 3. Ttl (cache V1)
```

Comment:
Some more detailed explanations about all three of these options would be useful. In particular we should specify for each one:

1. What is used as the cache key for each strategy.
2. What is involved in computing each key.
3. The limitations that each strategy possesses.

So that customers have some idea of what they're picking.